### DISCUSSION TITLE
Sorry AIM not gold English- Bug her

### DISCUSSION THREAD

**Discussion by carrabelloy** [community/community#142027] [Programming Help]
### Body

Null check operator used on a null value
#0      Element.widget (package:flutter/src/widgets/framework.dart:3484)
#1      Provider._inheritedElementOf (package:provider/src/provider.dart:343)
#2      Provider.of (package:provider/src/provider.dart:293)
#3      Matrix.of (package:fluffychat/widgets/matrix.dart:61)
#4      UserBottomSheetView.build (package:fluffychat/pages/user_bottom_sheet/user_bottom_sheet_view.dart:31)
#5      StatelessElement.build (package:flutter/src/widgets/framework.dart:5557)
#6      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5487)
#7      Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#8      StatelessElement.update (package:flutter/src/widgets/framework.dart:5563)
#9      Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#10     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#11     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
#12     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#13     StatefulElement.update (package:flutter/src/widgets/framework.dart:5673)
#14     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#15     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#16     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#17     ProxyElement.update (package:flutter/src/widgets/framework.dart:5816)
#18     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#19     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#20     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#21     ProxyElement.update (package:flutter/src/widgets/framework.dart:5816)
#22     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#23     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#24     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
#25     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#26     StatefulElement.update (package:flutter/src/widgets/framework.dart:5673)
#27     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#28     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776)
#29     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#30     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#31     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#32     ProxyElement.update (package:flutter/src/widgets/framework.dart:5816)
#33     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#34     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776)
#35     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#36     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#37     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#38     StatelessElement.update (package:flutter/src/widgets/framework.dart:5563)
#39     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#40     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776)
#41     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#42     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#43     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
#44     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#45     StatefulElement.update (package:flutter/src/widgets/framework.dart:5673)
#46     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#47     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#48     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
#49     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#50     StatefulElement.update (package:flutter/src/widgets/framework.dart:5673)
#51     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#52     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776)
#53     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#54     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776)
#55     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#56     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776)
#57     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#58     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#59     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
#60     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#61     StatefulElement.update (package:flutter/src/widgets/framework.dart:5673)
#62     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#63     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#64     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#65     StatelessElement.update (package:flutter/src/widgets/framework.dart:5563)
#66     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#67     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#68     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
#69     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#70     StatefulElement.update (package:flutter/src/widgets/framework.dart:5673)
#71     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#72     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776)
#73     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#74     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776)
#75     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#76     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776)
#77     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#78     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#79     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
#80     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#81     StatefulElement.update (package:flutter/src/widgets/framework.dart:5673)
#82     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#83     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#84     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
#85     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#86     StatefulElement.update (package:flutter/src/widgets/framework.dart:5673)
#87     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#88     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#89     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#90     StatelessElement.update (package:flutter/src/widgets/framework.dart:5563)
#91     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#92     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#93     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#94     ProxyElement.update (package:flutter/src/widgets/framework.dart:5816)
#95     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#96     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776)
#97     Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#98     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#99     Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#100    StatelessElement.update (package:flutter/src/widgets/framework.dart:5563)
#101    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#102    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#103    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#104    ProxyElement.update (package:flutter/src/widgets/framework.dart:5816)
#105    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#106    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#107    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#108    ProxyElement.update (package:flutter/src/widgets/framework.dart:5816)
#109    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#110    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#111    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#112    StatelessElement.update (package:flutter/src/widgets/framework.dart:5563)
#113    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#114    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#115    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#116    StatelessElement.update (package:flutter/src/widgets/framework.dart:5563)
#117    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#118    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776)
#119    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#120    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776)
#121    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#122    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#123    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
#124    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#125    StatefulElement.update (package:flutter/src/widgets/framework.dart:5673)
#126    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#127    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#128    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
#129    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#130    StatefulElement.update (package:flutter/src/widgets/framework.dart:5673)
#131    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#132    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776)
#133    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#134    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#135    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#136    ProxyElement.update (package:flutter/src/widgets/framework.dart:5816)
#137    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:105)
#138    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#139    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776)
#140    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#141    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#142    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
#143    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#144    StatefulElement.update (package:flutter/src/widgets/framework.dart:5673)
#145    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#146    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#147    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#148    ProxyElement.update (package:flutter/src/widgets/framework.dart:5816)
#149    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#150    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#151    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#152    ProxyElement.update (package:flutter/src/widgets/framework.dart:5816)
#153    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#154    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#155    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
#156    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#157    StatefulElement.update (package:flutter/src/widgets/framework.dart:5673)
#158    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#159    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#160    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#161    StatelessElement.update (package:flutter/src/widgets/framework.dart:5563)
#162    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#163    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#164    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#165    StatelessElement.update (package:flutter/src/widgets/framework.dart:5563)
#166    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#167    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6776)
#168    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#169    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#170    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#171    ProxyElement.update (package:flutter/src/widgets/framework.dart:5816)
#172    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#173    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#174    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#175    ProxyElement.update (package:flutter/src/widgets/framework.dart:5816)
#176    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#177    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#178    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
#179    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#180    StatefulElement.update (package:flutter/src/widgets/framework.dart:5673)
#181    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#182    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#183    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
#184    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#185    StatefulElement.update (package:flutter/src/widgets/framework.dart:5673)
#186    Element.updateChild (package:flutter/src/widgets/framework.dart:3827)
#187    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5512)
#188    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5650)
#189    Element.rebuild (package:flutter/src/widgets/framework.dart:5203)
#190    BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2905)
#191    WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:1136)
#192    RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:443)
#193    SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1392)
#194    SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1313)
#195    SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:1171)
#196    _invoke (dart:ui/hooks.dart:312)
#197    PlatformDispatcher._drawFrame (dart:ui/platform_dispatcher.dart:419)
#198    _drawFrame (dart:ui/hooks.dart:283)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by habeshacoder** [comment]
TBH, it is hard without more information.
